randomArray.js

```diff
@@ -0,0 +1,17 @@
+const _ = require("underscore")
+
+function mixedArray1 (arr) {
+  const newArr = _.shuffle(arr);
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] === newArr[i]){
+      newArr.splice(i, 1);
+      const randomIndex = Math.floor(Math.random() * newArr.length);
```

Comment:
Actually, picking a random element is ok.  

But, can we simplify the code in lines 10-11 by using element swapping instead.